tests/network/hotplug.go

```diff
@@ -306,6 +306,14 @@ var _ = SIGDescribe("nic-hotunplug", func() {
 			By("verify unplugged interface is not reported in the VMI status")
 			vmi = verifyDynamicInterfaceChange(vmi, plugMethod)
 
+			By("verify unplugged iface cleared from VMI spec")
+			Eventually(func(g Gomega) {
```

Comment:
I do not understand how this `Eventually` does its thing.
Nothing refreshes the `vmi` per what I see. Did I miss something?

Also, I am missing the `vm` object being checked.

Reply:
Thanks, you can resolve it now.